### ISSUE TITLE
Interactive elements not reachable with TalkBack and/or keyboard [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#248]
The "Find a site near you" link can be reached using TalkBack, but it is not actionable. 
When using the keyboard alone, the link cannot be reached or actioned.

![health ResultC - Link - Find a site near you](https://user-images.githubusercontent.com/7034772/92039363-f0b15780-ed42-11ea-9ea1-47a32c471950.png)

![health ResultC - Link - NYS Health Hotline](https://user-images.githubusercontent.com/7034772/92122490-7bd53080-edc9-11ea-9e98-63502177cee1.png)

**Remediation Guidance**
Ensure all actionable content is accessible using TalkBack and/or keyboard.

**Comment by floridemai** [comment]
@jonnyjames Can you check in the next build please. Thanks

**Comment by jonnyjames** [comment]
@floridemai The "Find a site near you" and the phone number links can be tapped using TalkBack alone, but when using both an external Bluetooth keyboard **and** TalkBack is running, the links receive focus but cannot be actioned.

**Comment by ckiss** [comment]
fixed, avail in next build

**Comment by jonnyjames** [comment]
This issue is now fixed. Closing this issue.
